src/app/store/keys/key.slice.ts

```diff
@@ -12,10 +13,12 @@ interface KeyConfigSoftware {
   encryptedSecretKey: string;
   salt: string;
 }
+
 interface KeyConfigLedger {
   type: 'ledger';
   id: string;
   publicKeys: StxAndIdentityPublicKeys[];
+  bitcoin?: Record<string, BitcoinLedgerAccountDetails>;
```

Comment:
This feels confusing to me, why `publicKeys` for stacks but bitcoin is just `bitcoin`?

Reply:
Good catch, removed. I started putting data here, then decided to create a new slice instead.